.circleci/config.yml

```diff
@@ -175,24 +178,28 @@ jobs:
     steps:
       - checkout
       - restore-cargo-cache
-      # Check this to make sure we do not include patched dependencies in
-      # the Cargo.lock.
+      # Check this to make sure we do not include patched dependencies in the Cargo.lock.
       - run:
           command: |
-            [[ -z $(cat Cargo.lock | grep "\[\[patch.unused\]\]") ]]
+            [ -z $(grep "\[\[patch.unused\]\]" Cargo.lock) ]
       - run: cargo fmt --all --check
       # TODO: this is incompatible with workspace inheritance, uncomment when
       # https://github.com/DevinR528/cargo-sort/pull/29 is merged
       # - run: cargo install cargo-sort
       # - run: cargo sort --check --workspace
-      - run: |
-          cargo clippy --tests \
-                       --all-targets \
-                       --all-features \
-                       --no-deps -- \
-                       --D warnings \
-                       -A clippy::let-unit-value \
-                       -A clippy::format-push-string
+      - run: cargo clippy --tests --all-targets --all-features --no-deps -- --D warnings
+      - save-cargo-cache
+  test-member:
+    parameters:
+      crate:
+        description: "Crate in workspace to test"
+        type: string
+    executor: docker-rust
+    steps:
+      - checkout
+      - restore-cargo-cache
+      - apply-patches
+      - run: cargo test --package << parameters.crate >> --all-features -- --nocapture
```

Comment:
How's this different than check-standalone?

Reply:
check-standalone does a full fmt+clippy+test, but when the crates are in the workspace fmt+clippy is already done, so test-member only tests a workspace member, unlike platform-test which also does integration tests with examples etc. I can clarify the names a bit.